PyRoute/TradeCodes.py

```diff
@@ -499,6 +499,13 @@ def is_well_formed(self):
             msg = "Sophont codes must be 5 chars each - got at least " + bad_sophonts[0]
             return False, msg
 
+        # explicitly exclude multiple W-pop (ie, 95+%) sophonts
+        big_sophs = [code for code in self.sophont_list if code.endswith('W')]
+        if 1 < len(big_sophs):
+            sophs = ' '.join(big_sophs)
+            msg = "Can have at most one W-pop sophont.  Have " + sophs
+            return False, msg
+
```

Comment:
Nothing in the TravMap data had two implicit-and-or-W sophont codes on the same star, so I felt fairly safe excluding it here.